core/domain/training_job_result_domain.py

```diff
@@ -0,0 +1,56 @@
+# coding: utf-8
+#
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Functions and classdefs related to protobuf files used in Oppia-ml"""
+
+from core.classifiers import algorithm_registry
+
+class TrainingJobResult(object):
+    """TrainingJobResult domain object."""
```

Comment:
What is a TrainingJobResult? This would be nice context to include here. It's clear what this is a domain object & what its name is from the code--what additional details can we provide here to provide context for how this class should be broadly used?

Reply:
Added description.